app/src/main/java/app/service/jurisdiction/JurisdictionService.java

```diff
@@ -56,7 +66,14 @@ public JurisdictionDTO createJurisdiction(CreateJurisdictionDTO requestDTO, Long
         jurisdiction.setPrimaryHoverColor(requestDTO.getPrimaryHoverColor());
         jurisdiction.setLogoMediaUrl(requestDTO.getLogoMediaUrl());
 
-        return new JurisdictionDTO(jurisdictionRepository.save(jurisdiction));
+        Jurisdiction savedJurisdiction = jurisdictionRepository.save(jurisdiction);
+
+        Set<LatLongDTO> dtoBounds = requestDTO.getBounds();
+        if (dtoBounds != null) {
```

Comment:
The bounds should be required.  We should also check and make sure that it is a valid polygon (that the first and last tuples are equal).

Reply:
Is a polygon bound valid if there are only two tuples and they're both the same?

Reply:
It seems like you need three noncolinear points.  I don't think we need to require that the first and last be the same, i.e., we can assume the final edge.